src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_UsingStatement.cs

```diff
@@ -474,15 +474,13 @@ private BoundExpression MakeCallWithNoExplicitArgument(SyntaxNode syntax, BoundE
                 ? ImmutableArray.Create(expression)
                 : ImmutableArray<BoundExpression>.Empty;
 
-            var refKinds = method.IsExtensionMethod && !method.ParameterRefKinds.IsDefaultOrEmpty
-                ? ImmutableArray.Create(method.ParameterRefKinds[0])
-                : default;
+            Debug.Assert(!method.IsExtensionMethod || method.ParameterRefKinds.IsDefaultOrEmpty || method.ParameterRefKinds[0] != RefKind.Ref);
 
             BoundExpression disposeCall = MakeCall(syntax: syntax,
                 rewrittenReceiver: receiver,
                 method: method,
                 rewrittenArguments: args,
-                argumentRefKindsOpt: refKinds,
+                argumentRefKindsOpt: default,
```

Comment:
What about the ref kind on `this`? For example, in `GetEnumerator(this in Collection c)`, don't we need to pass the `in` along here?

Reply:
The `in` is implicit, so the argument does not have a ref kind.
We only pass in the ref kind when the `in` is explicit. That emits slightly different code.